GliaWidgets/Localization+StringProviding.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+extension Localization {
+    static func tr(
+        _ table: String,
+        _ key: String,
+        _ args: CVarArg...,
+        fallback value: String,
+        stringProviding: StringProviding? = Glia.sharedInstance.stringProviding
+    ) -> String {
+        guard
+            let stringProviding,
+            let remoteString = stringProviding.getRemoteString(key)
+        else {
+            let format = BundleToken.bundle.localizedString(forKey: key, value: value, table: table)
+            return String(format: format, locale: Locale.current, arguments: args)
+        }
+
+        return remoteString
+    }
+}
+
+private final class BundleToken {
```

Comment:
I suppose we can replace this with existed [one](https://github.com/salemove/ios-sdk-widgets/blob/71d7bc71b5827bcfc04d5c5f0b1826c616204f4a/GliaWidgets/Sources/BundleManaging/BundleManaging.swift#L3).

Reply:
Indeed, fixed da02274a862e6161eaf7631d5007ca19b6067237